Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -335,6 +335,107 @@ public extension Tensor {
     static func ++ (lhs: Tensor, rhs: Tensor) -> Tensor {
         return lhs.concatenated(with: rhs)
     }
+
+    /// Gathers slices of this tensor at `indices` along the `axis` dimension.
```

Comment:
Doc comments for non-mutating methods often starts with `Returns a ... by <verb>ing ...` instead of `<Verb>s ...`. Quite a few doc comments in the library do not follow this guideline yet and should be updated.

Reply:
Sounds good. I'll try to go through them tomorrow.

Reply:
Filed #160 .